### PR TITLE
V abt/meeting app nodejs

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**
Meeting app in nodejs

### CODE REVIEW

File: samples/meeting-app/nodejs/ClientApp/src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (v-chetsh) [inline samples/meeting-app/nodejs/ClientApp/src/setupTests.js:5]: Is this file required?

File: samples/meeting-app/nodejs/api/server/bot/botActivityHandler.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    addConversationReference(activity) {
+        const conversationReference = TurnContext.getConversationReference(activity);
+        conversationReferences[conversationReference.conversation.id] = conversationReference;
+    }
+
+    // addConversationDataReference(activity) {
+    //     const conversationReference = TurnContext.getConversationReference(activity);
+    //     this.conversationReferences[conversationReference.conversation.id] = conversationReference;
```
Comment (v-chetsh) [inline samples/meeting-app/nodejs/api/server/bot/botActivityHandler.js:48]: Remove if not required.

**Review by v-chetsh — CHANGES_REQUESTED** [review]
Reviewed and provided my comments.

File: samples/meeting-app/nodejs/ClientApp/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
Comment (v-chetsh) [inline samples/meeting-app/nodejs/ClientApp/README.md:5]: Make sure to link it with root file.

File: samples/meeting-app/nodejs/api/.env [hunk 4]
```diff
@@ -0,0 +1,2 @@
+MicrosoftAppId=""
+MicrosoftAppPassword=""
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/api/.env:2]: not required here

File: samples/meeting-app/nodejs/ClientApp/public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/public/manifest.json:24]: It's react specific metadata.

File: samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/basic-details-mobile.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        };
+
+        microsoftTeams.tasks.startTask(taskInfo, (err, note) => {
+            if (err) {
+                console.log("Some error occurred in the task module")
+                return
+            }
+            const assetDetail: IAssetDetails = {
+                message: note
+            }
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/basic-details-mobile.tsx:34]: done

File: samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/questions.tsx [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+                        return {
+                            meetingId: context.meetingId!,
+                            question: question.value,
+                            setBy: context.userPrincipalName!,
+                            isDelete: 0
+                        };
+                    }
+                })
+
+
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/questions.tsx:54]: done

File: samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/questions.tsx [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+            width: 400,
+            url: `${window.location.origin}/edit?editText=` + editText,
+        };
+
+        microsoftTeams.tasks.startTask(taskInfo, (err, question: string) => {
+            if (err) {
+                console.log("Some error occurred in the task module")
+                return
+            }
+            microsoftTeams.getContext((context) => {
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/questions.tsx:81]: done

File: samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/timeline.tsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
+import { Flex, Card, Text, NotesIcon } from '@fluentui/react-northstar'
+import "../../recruiting-details/recruiting-details.css"
+
+const Timeline = () => {
+    const rowsPlain = [
+        {
+            key: 1,
+            items: [
+                {
+
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/src/components/recruiting-details/basic-details/timeline.tsx:10]: done

File: samples/meeting-app/nodejs/api/server/bot/botActivityHandler.js (hunk 2 above)
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/api/server/bot/botActivityHandler.js]: not required

File: samples/meeting-app/nodejs/ClientApp/src/setupTests.js (hunk 1 above)
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/src/setupTests.js:5]: just some test info, no issue to keep it.

File: samples/meeting-app/nodejs/ClientApp/README.md (hunk 3 above)
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/nodejs/ClientApp/README.md:5]: done

**Review by v-chetsh — APPROVED** [review]
Changes looks fine for me, approving it.

File: samples/meeting-recruitment-app/nodejs/Readme.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+   Please follow [View account keys](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal#view-account-access-keys) to see the   
+   keys info.
+
+3) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+4) We have two different solutions to run so follow below steps:
+ 
+   A) In a terminal, navigate to `samples/meeting-app/nodejs/api`
```
Comment (Wajeed-msft) [inline samples/meeting-recruitment-app/nodejs/Readme.md:37]: This path needs to be changed.

**Review by Wajeed-msft — APPROVED** [review]
